ci-operator/templates/cluster-launch-installer-e2e-smoke.yaml

```diff
@@ -16,8 +16,8 @@ parameters:
   required: true
 - name: IMAGE_TESTS
   required: true
-- name: IMAGE_INSTALLER_SMOKE
-  required: true
+- name: LOCAL_IMAGE_INSTALLER_SMOKE
+  value: ${NAMESPACE}/pipeline:installer-smoke
```

Comment:
I'm guessing there is a better way for this.. but this works for me when I call ci-operator and this template from my local machine

Reply:
I'm testing with this as `required: true` .. if everything is as expected this should work